.vscode/tasks.json

```diff
@@ -73,6 +73,17 @@
                 "-c",
                 "${workspaceFolder}/burstsvc/burst.tests/gen_coverage_report.sh"], // 
             "problemMatcher": ["$msCompile"]
+        },
+        {
+            "label": "SubTask-Run Checkov image",
+            "type": "shell",
+            "command": "docker run -it -v /workspaces/istio:/istio --rm bridgecrew/checkov -d /istio --output-file-path /istio --framework 'kubernetes' 'dockerfile' ; returncode=$?; mv results_cli.txt checkov_scan_results.txt; echo ' '; echo '-------------------------- Checkov Results Summary --------------------------';echo ' ';sed -n '/results:/{p;n;n;N;p;n;n;}' checkov_scan_results.txt;echo ' ';echo '-------------------------- Checkov Results Summary --------------------------';echo ' '; code -r ${workspaceFolder}/checkov_scan_results.txt;echo Scan Complete - Please review 'checkov_scan_results.txt' for details.; exit $returncode"
```

Comment:
Same Comment as: https://github.com/retaildevcrews/loderunner/pull/283#discussion_r931616632

> By default, all results are printed and written to output.
> Should we add `--quiet --compact` to the arg?
> It produces smaller output with only FAILED results.
> ```
> --quiet               in case of CLI output, display only failed checks. Also disables progress bars
> --compact             in case of CLI output, do not display code block
> ```